colorcode.c

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
```

Comment:
File name maybe renamed in order to reflect that nature of contents, for ex colorCodeTest.c